src/Images.jsx

```diff
@@ -232,6 +232,28 @@ class Images extends React.Component {
 
         const imageRows = filtered.map(id => this.renderRow(this.props.images[id]));
 
+        const sortRows = (rows, direction, idx) => {
+            // Image / Owner / Created / ID / Disk space / Used by
+            const isNumeric = idx === 2 || idx === 4 || idx === 5;
+            const sortedRows = rows.sort((a, b) => {
+                let aitem = a.columns[idx].sortKey ?? a.columns[idx].title;
+                let bitem = b.columns[idx].sortKey ?? b.columns[idx].title;
+
+                // Use raw value for sorting
+                if (idx === 1 || idx === 2 || idx === 4 || idx === 5) {
+                    aitem = a.columns[idx].rawValue;
+                    bitem = b.columns[idx].rawValue;
+                }
```

Comment:
See, above, using `sortKey` would make this obsolete.

Reply:
deleted in https://github.com/cockpit-project/cockpit-podman/pull/2239/commits/47cb5ac2268d9e22b8636a9e41a6b0ee25463ee8